09LessonNine/in-class-project/css/style.css

```diff
@@ -0,0 +1,69 @@
+body{
+  font-family:sans-serif;
+}
+main {
+  background-image:url("../img/featured-image-4.jpg");
+  background-repeat: no-repeat;
+  background-position:center;
+  background-size:cover;
+  height:350px;
+  width:100%;
+  text-align: center;
+  padding-top:100px;
+  color:white;
+}
+section{
+  clear:both;
+  width:100%
+  height:500px;
+  padding-top:10px;
+}
+h1{
+  padding-top:75px;
+}
+.leftbox, .rightbox {
+  float:left;
+  color:white;
+  height:400px;
+  width:50%;
+  text-align:center;
+  margin-top:20px;
```

Comment:
I'd like you to finish this site out. Add the appropriate margin to these divs so this page looks exactly like the example, please.